### PR TITLE
Implement basic Pane rendering test

### PR DESCRIPTION
**Pull request by cherewaty**
Renders a `Pane` inside a `Paneset`.

### CODE REVIEW

File: lib/Pane/tests/interactor.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {
+  interactor,
+  isPresent
+} from '@bigtest/interactor';
+import css from '../Pane.css';
+
+export default interactor(class PaneInteractor {
+  panePresent = isPresent(`.${css.pane}`);
```
Comment (wwilsman) [inline lib/Pane/tests/interactor.js]: So we can make this interactor more reusable, we can do this:

``` javascript
export default interactor(class PaneInteractor {
  static defaultScope = `.${css.pane}`
});
```

And use the default `isPresent` property

``` javascript
expect(pane.isPresent).to.be.true
```

This way when we eventually test multiple panes, we can pass IDs (or data-test attributes) to scope multiple `PaneInteractor`s to their respective pane components

Comment (cherewaty) [reply lib/Pane/tests/interactor.js]: 💯 That's an excellent feature, Mr. @wwilsman.

File: lib/Pane/tests/Pane-test.js [hunk 2]
```diff
@@ -1 +1,25 @@
-import Pane from '../Pane'; // eslint-disable-line no-unused-vars
+import React from 'react';
+import { describe, beforeEach, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import { mountWithContext } from '../../../tests/helpers';
```
Comment (wwilsman) [inline lib/Pane/tests/Pane-test.js]: Is the context truly needed for a pane / paneset? 😢

Comment (cherewaty) [reply lib/Pane/tests/Pane-test.js]: Yes, because of the `closeItem` `aria-label` on a `PaneHeader` :(

**Pull request merged**